src/mono/mono/metadata/native-library.c

```diff
@@ -1238,8 +1238,10 @@ lookup_pinvoke_call_impl (MonoMethod *method, MonoLookupPInvokeStatus *status_ou
 	new_scope = g_strdup (orig_scope);
 	new_import = g_strdup (orig_import);
 #endif
-
 #ifdef ENABLE_NETCORE
+#ifndef HOST_WIN32
```

Comment:
You probably don't have to ifdef the label unless the compiler is actually throwing a warning about an unused label.

Reply:
It was throwing a warning, that is the reason why I put in the ifdef.

Reply:
Gotcha.